packages/firebase_dynamic_links/lib/src/firebase_dynamic_links.dart

```diff
@@ -48,6 +48,9 @@ class FirebaseDynamicLinks {
       Map<dynamic, dynamic> linkData) {
     if (linkData == null) return null;
 
+    final link = linkData['link'];
+    if (link == null) return null;
```

Comment:
info: if the `link` retrieved from [FIRDynamicLink#url](https://firebase.google.com/docs/reference/ios/firebasedynamiclinks/api/reference/Classes/FIRDynamicLink#/c:objc(cs)FIRDynamicLink(py)url) or [PendingDynamicLinkData#getUrl()](https://firebase.google.com/docs/reference/android/com/google/firebase/dynamiclinks/PendingDynamicLinkData#public-uri-getlink) is null we want to return null.